mobile/big.go

```diff
@@ -35,6 +35,16 @@ func NewBigInt(x int64) *BigInt {
 	return &BigInt{big.NewInt(x)}
 }
 
+// NewBigIntFromString allocates and returns a new BigInt set to x
+// interpreted in the provided base.
+func NewBigIntFromString(x string, base int) *BigInt {
+	b, success := (new big.Int).SetString(x, base)
```

Comment:
>>> build/cache/golangci-lint-1.27.0-linux-amd64/golangci-lint run --config .golangci.yml ./...

WARN [runner] Can't run linter goanalysis_metalinter: deadcode: analysis skipped: errors in package: [/home/travis/gopath/src/github.com/ethereum/go-ethereum/mobile/big.go:41:21: expected ')', found big] 

```
ERRO Running error: deadcode: analysis skipped: errors in package: [/home/travis/gopath/src/github.com/ethereum/go-ethereum/mobile/big.go:41:21: expected ')', found big] 


```

I guess it's new(big.Int).Set.

Sorry for fooling you...

Reply:
NP, will push a fix